File: Quick-annotations/src/main/java/com/wpf/app/quick/annotations/GroupView.java
package com.wpf.app.quick.annotations;

import androidx.annotation.IdRes;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by 王朋飞 on 2022/6/13.
 */
@Target(ElementType.FIELD)
@Retention(RetentionPolicy.RUNTIME)
public @interface GroupView {
    @IdRes int[] idList();
}

File: Quick-annotations/src/main/java/com/wpf/app/quick/annotations/bind/GroupView.kt
package com.wpf.app.quick.annotations.bind

import androidx.annotation.IdRes

/**
 * Created by 王朋飞 on 2022/6/13.
 */
@Target(AnnotationTarget.FIELD)
@Retention(AnnotationRetention.RUNTIME)
annotation class GroupView(@IdRes val idList: IntArray)